src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const twin = {};
+
+  // Iterate through robot's keys and copy all props to twin
+  for (const key in robot) {
+    twin[robot[key]] = key;
+  }
+
+  const allKeys = Object.keys(robot);
+  const uniqueKeys = Object.keys(twin);
```

Comment:
you can just check twin.hasOwnProprty(robot[key]) in the loop above before you saved reversed values into a twin.

Reply:
thanks for the tip, now code looks neat!